packages/cozy-sharing/src/components/ShareModal.jsx

```diff
@@ -93,7 +93,7 @@ ShareModal.propTypes = {
   hasSharedChild: PropTypes.bool,
   hasSharedParent: PropTypes.bool,
   isOwner: PropTypes.bool,
-  link: PropTypes.string.isRequired,
+  link: PropTypes.string,
```

Comment:
Doesn't seem to be always required, works fine in cozy-notes without it. Removed the required for less console spam

Reply:
well, it's null by default as I can see. Not too sure why it works on Notes without the prop, I could investigate

Reply:
I'm going to backtrack on that change, I'm not too sure what's supposed to happen with these props. But in any case, Notes never provided it in my use case.